packages/foundry-sdk-generator/src/generate/betaClient/__tests__/bundleDependencies.test.ts

```diff
@@ -280,22 +419,31 @@ export type C = D;`,
 
     const projectMinifier = new ProjectMinifier(
       project,
-      new Set(["A", "C"]),
-      "/internal/moduleA/index.ts",
+      new Set(["internal/moduleA:A", "internal/moduleA:C"]),
     );
     projectMinifier.minifyProject();
 
     expect(outputModule(project)).toMatchInlineSnapshot(`
       "/** /internal/moduleA/index.ts **/
       declare module "internal/moduleA" {
-      	export * from "internal/moduleB"
+      	export { A, C } from "internal/moduleB"
       }
       /** /internal/moduleB/index.ts **/
       declare module "internal/moduleB" {
       	export const A : {};
       	import * as B from "internal/moduleB/folder";
       	export type C = B.C;
       }
+      /** /internal/moduleB/folder/fileB.ts **/
+      declare module "internal/moduleB/folder/fileB" {
+      	export type B = string; export type Unused = string;
+      }
+      /** /internal/moduleB/folder/fileC.ts **/
+      declare module "internal/moduleB/folder/fileC" {
+      	import { B } from "internal/moduleB/folder/fileB";
+      	type D = B;
+      	export type C = D;
+      }
       /** /internal/moduleB/folder/index.ts **/
       declare module "internal/moduleB/folder" {
       	export type E = string;
```

Comment:
`import * as B` above should transitively keep the things that are `export *` from here, thus the `fileB` and `fileC` are required.